docs/self-managed/platform-deployment/helm-kubernetes/platforms/amazon-eks/eks-helm.mdx

```diff
@@ -0,0 +1,360 @@
+---
+id: eks-helm
+title: "EKS Helm Setup"
+description: "Guide on setting up required resources with Helm"
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+## Prerequisites
+
+- a Kubernetes Cluster, see [eksctl](#) or [terraform](#) guide <!-- TODO: replace with proper links -->
+- [Helm (3.13+)](https://helm.sh/docs/intro/install/) installed
+- [kubectl (1.28+)](https://kubernetes.io/docs/tasks/tools/#kubectl) to interact with the cluster.
+- (optional) Domain name / [hosted zone](https://docs.aws.amazon.com/Route53/latest/DeveloperGuide/hosted-zones-working-with.html) in Route53
+  - this allows to expose Camunda 8 and easily connect via [zbctl](../../../../../../apis-tools/cli-client/) or the [Camunda Modeler](https://camunda.com/download/modeler/)
+
+## Considerations
+
+### Architecture
+
+![Camunda 8 Self-Managed AWS Architecture Diagram](./assets/camunda-8-self-managed-architecture-aws.png)
+
+## Usage
+
+### Environment Prerequisites
+
+In order to streamline the execution of the subsequent commands, it is recommended to export multiple environment variables.
+
+The following are the required environment variables with some example values.
+
+```shell
+# Your standard region that you host AWS resources in
+export REGION=eu-central-1
+# Following two environment variables can be skipped if you don't have a domain
+# The domain name that you intend to use
+export DOMAIN_NAME=camunda.example.com
+# The e-mail to register with Let's Encrypt
+export MAIL=admin@camunda.example.com
+# The Ingress Helm Chart version
```

Comment:
Let's explicitly specify which one we use:
```suggestion
# The Ingress-Nginx Helm Chart version
export INGRESS_NGINX_HELM_CHART_VERSION="4.8.3"
```

Reply:
let's also quote the version so Docusaurus highlight is not confused (see the patch versions off-color):

<img width="411" alt="Screenshot 2023-11-29 at 15 46 31" src="https://github.com/camunda/camunda-docs/assets/6655714/d805bec7-abf5-418d-8442-416e171c20b8">